app/mixins/event-wizard.js

```diff
@@ -98,7 +98,7 @@ export default Mixin.create(MutableArray, CustomFormMixin, {
 
       // model.tax is set in basic-detail step to workaround issue #4385
       let tax = this.model.tax || data.tax;
-      if (tax && tax.name) {
+      if (tax) {
```

Comment:
don't do this

Reply:
Is there any specific reason??

Reply:
Yes there is, when something in tax will break we will see.